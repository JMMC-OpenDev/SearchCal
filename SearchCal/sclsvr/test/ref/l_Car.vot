version https://git-lfs.github.com/spec/v1
oid sha256:498ddf8c8d80f0a80f0497d5a8e85e2d91e0dd16a4e4af8b9847bd6de160f9e3
size 54836
