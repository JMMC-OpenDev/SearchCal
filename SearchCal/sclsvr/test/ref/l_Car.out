version https://git-lfs.github.com/spec/v1
oid sha256:d059a035cae3f2fa063105ee4c21e634c58da880d71e8d10abd3c21ab9007902
size 253845
